docs/high/group.rst

```diff
@@ -195,6 +195,19 @@ Reference
         Create a new link, or automatically create a dataset.
         See :ref:`group_links`.
 
+    .. method:: __bool__()
+
+        Check that the file the group belongs to is open:
```

Comment:
To be very picky, the same thing you said for datasets also applies to groups - a group also has an HDF5 ID which can be closed. I'm not sure there's any particular reason to do that explicitly when working with h5py, but we may as well be precise.

Other than that, this LGTM. :-)

Reply:
Ah, of course. I hadn't tried that.